### PR TITLE
Supervisor Subject Edit & Supervisor Subject Delete

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

File: app/views/supervisor/subjects/edit.html.erb [hunk 1]
```diff
@@ -1,2 +1,24 @@
-<h1>Subjects#edit</h1>
-<p>Find me in app/views/subjects/edit.html.erb</p>
+<h2><%= t :edit_subject %></h2>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for [:supervisor, @subject] do |subjects_form_obj| %>
```
Comment (chuanhtuan) [inline app/views/supervisor/subjects/edit.html.erb]: <%= form_for [:supervisor, @subject] do |f| %>

File: app/views/supervisor/subjects/edit.html.erb [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+      <%= subjects_form_obj.label :name %>
+      <%= subjects_form_obj.text_field :name, class: "form-control" %>
+
+      <%= subjects_form_obj.label :description %>
+      <%= subjects_form_obj.text_area :description, class: "form-control" %>
+
+      <h3><%= t :tasks %></h3>
+
+      <%= subjects_form_obj.fields_for :tasks do |tasks_form_obj| %>
```
Comment (chuanhtuan) [inline app/views/supervisor/subjects/edit.html.erb]: <%= f.fields_for :tasks do |builder| %>

**Comment by noor-framgia** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
Ok

**Comment by noor-framgia** [comment]
ok

**Pull request merged**